src/platform/packages/shared/kbn-discover-utils/src/data_types/logs/utils/get_all_logs_data_view_spec.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ALL_LOGS_DATA_VIEW_ID = 'discover-observability-root-profile-all-logs';
```

Comment:
Since this hasn't released yet, I wonder if we should take the opportunity to change this ID to something more generic like `discover-observability-solution-all-logs`?

Reply:
Sounds good, we're not using the constant yet so safe to rename

Reply:
Great, updated here: https://github.com/elastic/kibana/pull/209042/commits/2b01ea05db8e44eabc7cb1007cdeaf19bd3cd53e.